src/Features/VisualBasic/Portable/CodeFixes/Suppression/VisualBasicSuppressionCodeFixProvider.vb

```diff
@@ -129,10 +129,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.Suppression
             Return token.Kind = SyntaxKind.EndOfFileToken
```

Comment:
did the VB side of this as well.

Reply:
note: it's likely that all these fixes could just share all their code (using the IDE's existing built in generate-C#-or-VB functionality).  It's not clear ot me why they're separate fixers for everything.